cli/flox/src/commands/mod.rs

```diff
@@ -1070,6 +1071,10 @@ enum InternalCommands {
         #[bpaf(external(check_for_upgrades::check_for_upgrades))]
         check_for_upgrades::CheckForUpgrades,
     ),
+
+    /// Print information how to exit environment
+    #[bpaf(command, hide)]
```

Comment:
We should alias this to `deactivate` too for symmetry with `activate` and because that's what people haven't frequently looked for.